theano/gpuarray/opt.py

```diff
@@ -51,6 +51,7 @@
 
 
 gpu_optimizer = EquilibriumDB()
+gpu_optimizer2 = EquilibriumDB()
```

Comment:
It is because we register a different optimization that don't have the op_lifter wrapper.